### PR TITLE
Elaborate on using Vault + third party solutions

### PR DESCRIPTION
**Pull request by zaharidichev**
After a bit of reading, seems using cert manager + vault is the sensible way to do automatie cert rotation. I think explaining that + pointing to the docs + explaining what are the requirements to achieve auto cert rotation by our identitiy service makes sense.

Signed-off-by: Zahari Dichev <zaharidichev@gmail.com>

### CODE REVIEW

File: linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md [hunk 1]
```diff
@@ -10,33 +10,37 @@ an issuer certificate and private key. While Linkerd automatically rotates the
...
 how to automatically rotate the issuer certificate and private key, by using
-the *cert-manager* project.
+an external solution.
 
 (Note that Linkerd's trust anchor [must still be manually
 rotated](/2/tasks/manually-rotating-control-plane-tls-credentials/) on
 long-lived clusters.)
 
-## Installing cert-manager and create the Linkerd namespace
+## Using Cert manager
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md]: How about just `cert-manager`?

File: linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md [hunk 2]
```diff
@@ -114,13 +118,31 @@ kubectl get secret linkerd-identity-issuer -o yaml -n linkerd
 
 Now we just need to inform Linkerd to consume these credentials.
 
+### Cert manager with Vault
```
Comment (grampelberg) [inline linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md]: Just `Vault` with a TOC at the top (like the ingress doc)?

Comment (grampelberg) [reply linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md]: This section should just be instructions on how to use vault?

Comment (zaharidichev) [reply linkerd.io/content/2/tasks/automatically-rotating-control-plane-tls-credentials.md]: Ultimatelly using Vault is using Vault + cert manager...

**Pull request merged**